ReleaseNotes.md

```diff
@@ -49,6 +49,8 @@ For a complete list of Fixes and Improvements see the [Version 2.0](https://issu
 * [Fork on GitHub](https://github.com/fluid-project/infusion)
 * [Download a Build](https://github.com/fluid-project/infusion/releases)
 * [NPM](https://www.npmjs.com/package/infusion)
+* [Use from the CDNJS Content Distribution Network](https://cdnjs.com/libraries/infusion)
```

Comment:
Could you do some quick clean up here to make the ReleaseNotes match the Readme. That is adjust the ordering and link text for the various ways of getting Infusion.